src/configuration-factory.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  Flag,
+  HybridConfigurationStore,
+  IConfigurationStore,
+  MemoryOnlyConfigurationStore,
+  MemoryStore,
+} from '@eppo/js-client-sdk-common';
+
+import { LocalStorageBackedAsyncStore } from './local-storage';
+
+export function configurationStorageFactory(): IConfigurationStore<Flag> {
+  if (hasWindowLocalStorage()) {
```

Comment:
* chrome storage will be added here
* user provided persistence store added here too

testing is a bit tricky here owing to the `window` environment. got any ideas?

Reply:
Seems like folks just mock it ([StackOverflow link](https://stackoverflow.com/a/14381941/26197))